modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/PrepareMasterDatabaseForRelease.pm

```diff
@@ -236,23 +236,30 @@ sub pipeline_analyses_prep_master_db_for_release {
                 'db_conn'     => '#master_db#',
                 'input_query' => 'UPDATE method_link_species_set SET url = "" WHERE source = "ensembl"',
             },
-            -flow_into  => [ 'hc_master' ],
+            -flow_into  => [ 'master_db_name' ],
         },
 
-        {   -logic_name      => 'hc_master',
-            -module          => 'Bio::EnsEMBL::Compara::RunnableDB::RunJavaHealthCheck',
-            -parameters      => {
+        {   -logic_name => 'master_db_name',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::GetComparaDBName',
+            -parameters => {
                 'compara_db'  => '#master_db#',
-                'work_dir'    => $self->o('work_dir'),
-                'testgroup'   => 'ComparaMaster',
-                'output_file' => '#work_dir#/healthcheck.#testgroup#.out',
-                'ensj_conf'   => $self->o('ensj_conf'),
-                'run_healthchecks_exe' => $self->o('run_healthchecks_exe'),
-                'ensj_testrunner_exe'  => $self->o('ensj_testrunner_exe'),
             },
-            -flow_into       => [ 'backup_master' ],
-            -rc_name         => '2Gb_job',
-            -max_retry_count => 0,
+            -flow_into  => [ 'dc_master' ],
+        },
+
+        {   -logic_name        => 'dc_master',
+            -module            => 'Bio::EnsEMBL::DataCheck::Pipeline::RunDataChecks',
+            -parameters        => {
+                'datacheck_groups' => ['compara_master'],
+                'work_dir'         => $self->o('work_dir'),
+                'history_file'     => '#work_dir#/datacheck.compara_master.log',
```

Comment:
I don't see the history file like a normal log file useful for human consumption.
In https://www.ebi.ac.uk/seqdb/confluence/display/GTI/Everything+you+wanted+to+know+about+datachecks%2C+but+were+too+afraid+to+ask they say that all the shared history files found under `/nfs/panda/ensembl/production/datachecks/history` are `*.json`, so I would do the same here.

That said, the Runnable has an `output_file` option that has the TAP output of the tests. I think that would be really useful to keep around, esp. if we somehow want to make Jira tickets from it. I'm not very inspired, so I'm naming it `.tap.txt`, but `.log` would do.

```suggestion
                'history_file'     => '#work_dir#/datacheck.compara_master.history.json',
                'output_file'      => '#work_dir#/datacheck.compara_master.tap.txt',
```

Reply:
For HCs we used the regex `<hc_name>.<stage/db>.txt` (in case we want to keep the same one).